config.json

```diff
@@ -1,3 +1,13 @@
 {
-    "IP": "192.168.0.106"
+    "IP": "",
+    "EventNotificationChannelIdForShiny": "",
+    "EventNotificationChannelIdForBrilliant": "",
+    "EventNotificationChannelIdForMark": "",
+    "EventNotificationChannelIdForRareMark": "",
+    "NotificationChannelForInfo": "",
+    "TextNotificationChannelIdForShiny": "",
+    "TextNotificationChannelIdForBrilliant": "",
+    "TextNotificationChannelIdForMark": "",
+    "TextNotificationChannelIdForRareMark": "",
+    "EnableDebugLogging": "False"
```

Comment:
possibly add token here, and commands to change and save the config on the fly would avoid needing to edit more files

Reply:
Token added and test updated to accept it, so all config is now in config.json. I'll create a "Possible Future Improvements" section at the end of README.md and start recording things like "update config from discord on the fly" to it.